CreeDictionary/API/schema.py

```diff
@@ -54,6 +62,9 @@ class SerializedSearchResult(TypedDict):
     # Optional: we might not have some preverbs in our database
     preverbs: Tuple[Union[str, SerializedWordform], ...]
```

Comment:
Should we consider calling preverbs prefixes? (for the sake of linguistic completeness). Of course, person-prefixes, reduplicative prefixes and initial change are not included in preverbs (at least in our definition).